Wangscape/tilegen/TilesetGenerator.cpp

```diff
@@ -13,8 +13,18 @@ namespace tilegen
 TilesetGenerator::TilesetGenerator(const Options& options,
                                    std::unique_ptr<partition::TilePartitionerBase> tile_partitioner) :
     options(options),
-    mTilePartitioner(std::move(tile_partitioner))
+    mTilePartitioner(std::move(tile_partitioner)),
+    mDebugTileBuilder(mDebugTile, {0., 0., 1., 1.})
+    // TODO (Serin) this sf::Rect is repeated in TilePartitionerNoise, factor it out
```

Comment:
Here: https://github.com/Wangscape/Wangscape/blob/master/Wangscape/tilegen/partition/TilePartitionerNoise.cpp#L141. It defines the region where noise values are evaluated.